article-analyzer/src/main/java/co/inventorsoft/academy/spring/config/HelperWordsProperties.java

```diff
@@ -0,0 +1,16 @@
+package co.inventorsoft.academy.spring.config;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@Component
+@ConfigurationProperties(prefix = "helper")
```

Comment:
yep, another way to get properties, nice

Reply:
Heah, it's a little bit more convenient